t/22-svirt.t

```diff
@@ -234,4 +235,479 @@ subtest 'Method backend::svirt::open_serial_console_via_ssh()' => sub {
     is(shift @deleted_logs, $expected_serial_file, "Check if $expected_serial_file was deleted on die()");
 };
 
+subtest 'Method consoles::sshVirtsh::add_disk()' => sub {
+    my $ssh_creds_svirt = {hostname => 'hostname_svirt', password => 'password_svirt', username => 'root'};
+    my @last_ssh_commands;
+    my @last_ssh_args;
+    my @ssh_cmd_return;
+    my $_10gb = 1024 * 1024 * 1024 * 10;
+
+    my $mock_baseclass = Test::MockModule->new('backend::baseclass');
+    $mock_baseclass->redefine('run_ssh_cmd' => sub {
+            my ($self, $cmd, %args) = @_;
+            push @last_ssh_commands, $cmd;
+            push @last_ssh_args,     [%args];
+
+            my $ret = shift @ssh_cmd_return;
+            return undef unless defined $ret;
+
+            if (ref($ret) eq 'ARRAY') {
+                return @$ret;
+            } else {
+                return $ret;
+            }
```

Comment:
```suggestion
            return ref($ret) eq 'ARRAY' ? @$ret : $ret;
```

Reply:
I find more readable the multiline part. Sorry, I would prefer to keep it that way. The same applies above.